run_configs/_rose_all/suite.rc

```diff
@@ -0,0 +1,89 @@
+[scheduling]
+    [[dependencies]]
+        {% for PYTHON_VERSION in PYTHON_VERSIONS %}
+            {% for COMPILER in COMPILERS %}
```

Comment:
Rather than using Jinja2 looping you might find that [parameterised tasks](https://cylc.github.io/cylc-doc/7.8.8/html/suite-config.html#parameterized-tasks) give cleaner syntax.

Reply:
Oh that's so much better, thank you.